templates/patterns/footer/pattern-footer.html.twig

```diff
@@ -0,0 +1,39 @@
+<footer>
```

Comment:
the container from page does the job, this footer tag is redundant and so is below container